client/pages/cart.jsx

```diff
@@ -46,9 +51,30 @@ export default class Cart extends React.Component {
     })
       .then(res => res.json())
       .then(data => {
-        this.props.resetCart();
+        const orderId = data.orderId;
+        const emailElement = React.createElement(Email, { orderId, items });
+        const content = ReactDOMServer.renderToStaticMarkup(emailElement);
+
+        axios({
+          method: 'POST',
+          url: 'http://localhost:3001/send',
+          data: {
+            name: name,
+            email: email,
+            orderId: orderId,
+            html: content
+          }
+        }).then(res => {
+          if (res.data.msg === 'success') {
+            this.resetForm();
+          } else if (res.data.msg === 'fail') {
+            console.error('Failed to send confirmation email');
+          }
+        });
+
         window.location.hash = `order?orderId=${data.orderId}`;
```

Comment:
You should move this hash change into your `.then()` callback so that it waits for the order to complete.